src/funit/asserts/Assert_Complex.tmpl

```diff
@@ -94,9 +94,15 @@
 #include "unused_dummy.fh"
 
 module pf_AssertComplex_{rank}d
+#ifdef _ISO_REAL32
```

Comment:
I don't think these conditionals are necessary.  The constants are always defined.  They simply have the value `-1` if the compiler does not support that kind.